.github/workflows/action_test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test Action
+on: [push, pull_request]
+
+jobs:
+  export-secrets:
+    name: Test action
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out this repo
+      uses: actions/checkout@v2
+    - name: Run action
+      uses: ./
+      with:
+        gpg-priv-key: ''
+        gpg-priv-key-file: tests/keys/github-actions.priv
+        secrets-file: tests/data/example-secrets.yml
+    - name: Test Secrets from env
+      run: |
+        set -xe
+        echo 1
```

Comment:
Is this just for test?

Reply:
This is because _any_ occurance of the value in github logs should be masked. This is a manual test, but yes a test.

Reply:
Example output from test action job:

```
+ echo '*** should be masked'
*** should be masked
+ echo '*** should be masked'
*** should be masked
+ echo '*** should be masked'
*** should be masked
+ [[ *** == *** ]]
should see this
+ [[ *** == 0 ]]
+ echo 'should see this'
+ [[ *** == *** ]]
```